django_prometheus/middleware.py

```diff
@@ -146,6 +150,16 @@ def process_response(self, request, response):
         if hasattr(request, 'prometheus_after_middleware_event'):
             requests_latency.observe(TimeSince(
                 request.prometheus_after_middleware_event))
+            if hasattr(request, 'resolver_match'):
```

Comment:
Can you extract this logic to a separate function, to avoid duplication with ll. 132-135?